### DISCUSSION TITLE
Colspan for DataGridCell

### DISCUSSION THREAD

**Discussion by rpresb** [twilio-labs/paste#2031] [General] [answered]
In the footer I need to span the first column that contains the text `Total` to the first 4 columns.
![image](https://user-images.githubusercontent.com/3234769/141272515-df9a45f6-7602-4723-bdeb-29bc2caff1de.png)

In my case, the 1st column space is enough, but if my text was longer the text would make the column wider.
![image](https://user-images.githubusercontent.com/3234769/141273113-0ae8beb2-86bd-4cd7-9890-7e8cdec680e3.png)

**Comment by gloriliale** [comment]
Hi @rpresb! Thanks for this feedback on Data Grid, adding a colspan sounds very reasonable to us! Is there a specific timeline that you're looking to have this by? I've added [a ticket to make this happen](https://issues.corp.twilio.com/browse/DSYS-3136) in our backlog and we'll update this discussion w/ the progress on it.

**Reply by rpresb** [reply]
Thanks for the answer. I have no urgent need for it as I managed to fit everything I need for now.

**Comment by ghost** [comment, marked as answer]
@rpresb Hi there, just following up this prop should be available here for you to use: https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/data-grid/src/DataGridCell.tsx#L17
